wasm/src/project_system.rs

```diff
@@ -399,6 +399,7 @@ impl From<PackageInfo> for qsc_project::PackageInfo {
 pub(crate) fn into_qsc_args(
     program: ProgramConfig,
     entry: Option<String>,
+    ignore_dependency_errors: bool,
```

Comment:
ignore_dependency_errors is kind of a negation built into variable name. Consider fail_on_dependency_errors or something similar instead.

Reply:
I was thinking about this in terms of the "default" value being `false`, even though Rust doesn't actually have default values, so that you'd get the usual behavior passing `false` in, and would get the updated behavior by passing `true`. In this case the updated behavior is a suppression of errors. I can see what you mean by it being a bit of a negation in that you pass in `true` to make it do less.